doc/manual/manual.tex

```diff
@@ -7101,6 +7101,73 @@ \subsubsection{Simple convection in a spherical 3d shell}
 with a more elaborate material model we should be able to get even closer to
 reality.
 
+\subsubsection{Postprocessing spherical 3D convection}
+\label{sec:cookbooks-shell_3d_postprocess.prm}
+\textit{This section was contributed by Jacqueline Austermann, Ian Rose and Shangxin Liu}
+
+
+There are several postprocessors that can be used to turn the the velocity 
+and pressure solution into quantities that can be compared to surface 
+observations. In this cookbook we introduce two postprocessors: dynamic 
+topography and the geoid. We initialize the model with a harmonic 
+perturbation of degree 4 and order 2 and calculate the
+instantaneous solution. Analogous to the previous setup we use a spherical
+shell geometry model and a simple material model. 
```

Comment:
Can you add a sentence somewhere that provides the path to the complete .prm file, as is done in most other cookbooks?